src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -604,7 +604,8 @@ def __init__(
         self,
         backbone: Literal["tiny_vit", "vit_b"],
         num_classes: int = 0,
-        root_reference_info: Path | str = "vpm_zsl_reference_infos",
+        reference_info_dir: Path | str = "reference_infos",
+        infer_reference_info_root: Path | str = "../.latest/train",
```

Comment:
`.latest` is the output folder, which is only available in the CLI. This default value seems to need to be modified for API usage.
Also, since `.latest` is a soft-linked folder, it seems appropriate to browse to the actual folder.

Reply:
How can I access `work_dir` for training during test?
I want to save the files in `work_dir` for training and use them during test.
As far as I know, it seems there are only hard-coded path like that or user input like below
https://github.com/openvinotoolkit/training_extensions/blob/9b1c5c781efd70ad260fc1be4350b144bab66b92/tests/integration/api/test_auto_configuration.py#L47
https://github.com/openvinotoolkit/training_extensions/blob/9b1c5c781efd70ad260fc1be4350b144bab66b92/tests/integration/cli/test_cli.py#L107-L108